src/ingest-pipeline/airflow/dags/utils.py

```diff
@@ -1238,6 +1238,218 @@ def send_status_msg(**kwargs) -> bool:
     return send_status_msg
 
 
+def make_send_status_msg_function(
+        dag_file: str,
+        retcode_ops: List[str],
+        cwl_workflows: List[Path],
+        http_conn_id: str = 'entity_api_connection',
```

Comment:
It looks like it will cause a problem if http_conn_id is anything other than entity_api_connection now.  Maybe it's time for the whole argument to go away?

Reply:
looks like it is time. I'll remove it.